pliers/extractors/api/awsReko_test.py

```diff
@@ -0,0 +1,82 @@
+'''
+Extractors that interact with the AWS Rekognition API.
+'''
+
+from pliers.stimuli.image import ImageStim
+from pliers.extractors.base import Extractor, ExtractorResult
+import numpy as np
+import pandas as pd
+import boto3
+from pliers.utils import attempt_to_import, verify_dependencies
+
+aws_rekognition_client= attempt_to_import('boto3')
+
+
+class AwsRekognitionExtractor():
+    
+    def __init__(self,profile_name='default', region_name='us-east-1' ):
```

Comment:
`profile_name` and `region_name` should probably both be `None` by default, because both can be preconfigured in a credentials file and/or environment variables.